cmake/nuttx/px4_impl_nuttx.cmake

```diff
@@ -523,6 +532,17 @@ function(px4_os_add_flags)
 
 	set(added_exe_linker_flags) # none currently
 
+
```

Comment:
Good catch. I have no idea how that happened. It was definitely working in my branch, but I didn't test it after it went into master.